src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -19,20 +19,51 @@
     <UnsupportedPlatformsItems Include="$(UnsupportedPlatforms)"/>
   </ItemGroup>
 
+  <PropertyGroup>
+    <TestWithCore Condition="'$(TestWithCore)' == ''">true</TestWithCore>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TestWithCore)' == 'true'">
+    <TestRuntimeProjectJson Condition="'$(TestRuntimeProjectJson)' == ''">$(MSBuildThisFileDirectory)test-runtime\project.json</TestRuntimeProjectJson>
+    <TestRuntimeProjectLockJson Condition="'$(TestRuntimeProjectLockJson)' == ''">$(MSBuildThisFileDirectory)test-runtime\project.lock.json</TestRuntimeProjectLockJson>
+    <TestHost>corerun.exe</TestHost>
+    <XunitExecutable Condition="'$(XunitExecutable)' == ''">xunit.console.netcore.exe</XunitExecutable>
+    <DebugTestFrameworkFolder>dnxcore50</DebugTestFrameworkFolder>
+    <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TestWithCore)' != 'true'">
+    <XunitExecutable Condition="'$(XunitExecutable)' == ''">xunit.console.exe</XunitExecutable>
+    <DebugTestFrameworkFolder>dnx45</DebugTestFrameworkFolder>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TestWithCore)' == 'true'">
+    <TestTargetFramework Include="DNXCore,Version=v5.0">
+      <Folder>dnxcore50</Folder>
+    </TestTargetFramework>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TestWithCore)' != 'true'">
+    <TestTargetFramework Include="DNX,Version=v4.5">
```

Comment:
Should this be .NETFramework,Version=v4.5 and net45? We aren't set up to actually use DNX. Even the DNXCore part above will change I believe.

/cc @ericstj